bodhi/server/services/updates.py

```diff
@@ -200,10 +200,10 @@ def set_request(request):
     try:
         update.set_request(request.db, action, request.user.name)
     except BodhiException as e:
-        log.exception("Failed to set the request")
+        log.error("Failed to set the request")
```

Comment:
We should retain the error message from ```e```:

```suggestion
        log.error("Failed to set the request: %s", e)
```